.github/workflows/close-inactive-bot.yml

```diff
@@ -17,6 +17,7 @@ jobs:
           stale-issue-label: 'stale'
           stale-issue-message: 'This issue is stale because it has been open for 30 days with no activity.'
           close-issue-message: 'This issue was closed because it has been inactive for 14 days since being marked as stale.'
+          exempt-issue-labels: 'no-stale'
```

Comment:
```suggestion
          remove-issue-stale-when-updated: true
          exempt-all-milestones: true
          exempt-all-issue-assignees: true
          exempt-all-issue-milestones: true
```

Reply:
we should also update to `v7`
and maybe bump the days before stale to 60

Reply:
updated to 7, changed days to 50, 60 seemed a too long to me